.travis.yml

```diff
@@ -29,17 +29,17 @@ env:
       COVERAGE=true
     # This environment tests the oldest supported anaconda env
     - DISTRIB="conda" PYTHON_VERSION="2.7" INSTALL_MKL="false"
-      NUMPY_VERSION="1.6.2" SCIPY_VERSION="0.11.0" CYTHON_VERSION="0.23"
+      NUMPY_VERSION="1.8.1" SCIPY_VERSION="0.13.3" CYTHON_VERSION="0.23"
       COVERAGE=true
     # This environment tests the newest supported anaconda env
     # It also runs tests requiring Pandas.
     - DISTRIB="conda" PYTHON_VERSION="3.6" INSTALL_MKL="true"
-      NUMPY_VERSION="1.11.2" SCIPY_VERSION="0.18.1" PANDAS_VERSION="0.19.1"
+      NUMPY_VERSION="1.12.1" SCIPY_VERSION="0.18.1" PANDAS_VERSION="0.19.1"
```

Comment:
Looks like `PANDAS_VERSION="0.19.2"` should fix the dependency conflict on Travis.